modules/plugins/jboss-as-7/src/main/java/org/rhq/modules/plugins/jbossas7/JBossProductType.java

```diff
@@ -143,6 +144,8 @@ private static JBossProductType determineJBossProductTypeViaProductConfFile(File
                 productType = JBossProductType.BPMS;
             } else if (slot.equals("dv")) {
                 productType = JBossProductType.JDV;
+            } else if (slot.equals("wildfly-full")) {
```

Comment:
Hi, thanks for the patch. While you're right that they've added the "wildfly-full" as product.conf to Wildfly 10 (thus breaking a little bit our earlier detection code), this isn't quite enough. This will detect the Wildfly 10, but you'll receive a InvalidPluginConfiguration at some point since the validateServerAttributes() in BaseServerComponent will also fail:

```
[standalone@localhost:9990 /] read-attribute product-name
WildFly Full
[standalone@localhost:9990 /]
```

And this doesn't match "wildfly-full". Also, I did not check the other Wildfly versions if the web for example returns "wildfly-web" (and it if does, then adding support for Wildfly 10 should detect these as well correctly).